resources/authgear/templates/en/web/authflowv2/__page_frame.html

```diff
@@ -37,5 +37,6 @@
   <div class="widget">
     {{ template "page-content" . }}
     {{ template "authflowv2/__lockout.html" . }}
+    {{ template "authflowv2/__watermark.html" . }}
```

Comment:
__lockout.html should still be the last element to ensure it is the last element, so that it is on top by default.